src/TextField/TextField.jsx

```diff
@@ -24,6 +24,9 @@ function isValid(value) {
 const TextField = React.createClass({
 
   propTypes: {
+    /**
+     * @ignore
```

Comment:
Yes, leave `children` - hopefully most react devs know what it refers to. And supplying children as a prop on a self closing tag does actually works (!), even though that's not how it's meant to be used.